app/src/main/java/com/example/nidoqueue/activity/WelcomeActivity.java

```diff
@@ -1,60 +1,41 @@
 package com.example.nidoqueue.activity;
 
-import android.annotation.SuppressLint;
 import android.os.Bundle;
 import android.os.Handler;
-import android.provider.ContactsContract;
 import android.view.View;
 import android.widget.ArrayAdapter;
 import android.widget.Button;
 import android.widget.Toast;
 
 import com.example.nidoqueue.R;
 import com.example.nidoqueue.controller.ContextManager;
-import com.example.nidoqueue.model.Database;
 import com.example.nidoqueue.controller.RequestManager;
 import com.example.nidoqueue.model.User;
-import com.example.nidoqueue.model.UserProfileContent;
-import com.google.firebase.firestore.FirebaseFirestore;
-
-import java.util.ArrayList;
 
 public class WelcomeActivity extends AbstractActivity implements UserProfileAddFragment.OnFragmentInteractionListener {
     Button signUp, signIn, clickHere;
     ArrayAdapter<User> Adapter;
     String Message; // String that is used to display information for each experiment on click.
-    ArrayList<User> userList;
-    String android_id;
-    // get instances of RequestManager and ContextManager
-    private static final RequestManager requestManager = RequestManager.getInstance();
-    private static final ContextManager contextManager = ContextManager.getInstance();
-    private static final Database database = Database.getInstance();
     boolean doubleBackToExitPressedOnce = false;
 
+    // get instances of RequestManager and ContextManager
+    private static final RequestManager requestManager = RequestManager.getInstance();
+    private static final ContextManager contextManager = requestManager.getContextManager();
+//    private static final DatabaseManager DATABASE_MANAGER = requestManager.getDatabaseManager();
 
-    private View.OnClickListener SignIn = new View.OnClickListener() {
-        @Override
-        public void onClick(View v) {
-            requestManager.signIn();
-        }
-    };
-
-    private View.OnClickListener SignUp = new View.OnClickListener() {
-        @Override
-        public void onClick(View v) {
-            requestManager.signUp();
-        }
-    };
 
+    private View.OnClickListener SignIn = v -> requestManager.signIn();
+    private View.OnClickListener SignUp = v -> requestManager.signUp();
```

Comment:
We should discuss this as a group and to decide which format we will define our click listeners, because we would have to change all the activity files.